lib/Elastica/Query/Ids.php

```diff
@@ -77,7 +77,7 @@ public function setType($type)
             return $this;
         }
 
-        $this->_params['type'] = $type;
+        $this->_params['type'] = (array)$type;
```

Comment:
I think I still miss here something. `$this->_params['type']` is already a string, so your example does not apply I think.